lib/active_record/connection_adapters/sqlserver_adapter.rb

```diff
@@ -1 +0,0 @@
-require 'arjdbc/mssql'
```

Comment:
if sqlserver adapter has this file we should keep it as a potential alternative working with `adapter: sqlserver`

Reply:
Yeah, pretty much the only reason there needs to be changes to this repo is because of the Java source needing to live in here for compiling.

Reply:
okay, would hope if the fork is to be maintained that the .java source would move there.
fine for now but this is the way AR-JDBC exts used to work in the past and I do not see a reason why keep it here (except if the Java base class API would not be 'stable'). before doing that its okay to have the native part in as I see a few minor pieces we should maybe extract away (as it gets merged - hopefully before release).

Reply:
I don't remember exactly but I think I had chatted with @enebo about it and its not that easy to separate out the java classes to a separate compile step. I suppose I could look into doing that but at this point I just want to get this out so that we can officially support rails 5.0 and I can get our (Finalsite's) app upgraded. I hope 5.1 and 5.2 will require minimal effort so I'd like to get into those once we get this one into the wild and get other people testing it out.